hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/listener/DbNotificationListener.java

```diff
@@ -1217,7 +1251,7 @@ private void addNotificationLog(NotificationEvent event, ListenerEvent listenerE
         params.add(catName);
       }
 
-      s = "insert into \"NOTIFICATION_LOG\" (" + columns + ") VALUES (" + insertVal + ")";
+      String s = "insert into \"NOTIFICATION_LOG\" (" + columns + ") VALUES (" + insertVal + ")";
```

Comment:
The very most of this query is constant, so I'd create a constant for it like this:
insert into \"NOTIFICATION_LOG\" (\"NL_ID\", \"EVENT_ID\", ...\"MESSAGE_FORMAT\"%s) VALUES (?, ... ?%s) then replace only the two %s placeholders with the potential cat, db, table name, and their ? characters. This would simplify the code a lot.

Reply:
@miklosgergely Yes, for sure.  And I can take a look at that in a future refactoring, but this request is out of scope for this change which only affects the generation of the "Next Event ID."  Thanks for the review.